### PR TITLE
Feature: add cen bandwdith limits data source

### PR DESCRIPTION
**Pull request by GiantRice**
This data source provides CEN Inter Region Bandwidth Limits available to the user.
$TF_ACC=1 go test ./alicloud -v -run=TestAccAlicloudCenBandwidthLimitsDataSource_instance_id -timeout=300m      
=== RUN   TestAccAlicloudCenBandwidthLimitsDataSource_instance_id
--- PASS: TestAccAlicloudCenBandwidthLimitsDataSource_instance_id (46.46s)
PASS
ok  	github.com/terraform-providers/terraform-provider-alicloud/alicloud	46.510s

### CODE REVIEW

File: alicloud/data_source_alicloud_cen_bandwidth_limits.go [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+	var allCenBwLimits []cbn.CenInterRegionBandwidthLimit
+
+	instanceIds := make([]string, 0)
+	if v, ok := d.GetOk("instance_ids"); ok {
+		for _, vv := range v.([]interface{}) {
+			instanceIds = append(instanceIds, Trim(vv.(string)))
+		}
+	} else {
+		instanceIds = append(instanceIds, "")
+	}
```
Comment (xiaozhu36) [inline alicloud/data_source_alicloud_cen_bandwidth_limits.go]: I think line 70-72 are redundant. Please remove it.

File: alicloud/data_source_alicloud_cen_bandwidth_limits.go [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+	for _, cenBwLimit := range allCenBwLimits {
+		mapping := map[string]interface{}{
+			"instance_id":        cenBwLimit.CenId,
+			"local_region_id":    cenBwLimit.LocalRegionId,
+			"opposite_region_id": cenBwLimit.OppositeRegionId,
+			"status":             cenBwLimit.Status,
+			"bandwidth_limit":    cenBwLimit.BandwidthLimit,
+		}
+
+		id := cenBwLimit.CenId + ":" + cenBwLimit.LocalRegionId + ":" + cenBwLimit.OppositeRegionId
```
Comment (xiaozhu36) [inline alicloud/data_source_alicloud_cen_bandwidth_limits.go]: Please use COLON_SEPARATED instead ":"

File: alicloud/data_source_alicloud_cen_bandwidth_limits_test.go [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+const testAccCheckAlicloudCenInterRegionBandwidthLimitsDataSourceCenIdConfig = `
+provider "alicloud" {
+  alias = "bj"
+  region = "cn-beijing"
+}
+
+provider "alicloud" {
+  alias = "sh"
+  region = "cn-shanghai"
+}
```
Comment (xiaozhu36) [inline alicloud/data_source_alicloud_cen_bandwidth_limits_test.go]: Please set a region to non-China region

File: alicloud/data_source_alicloud_cen_bandwidth_limits.go [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+				Elem:     &schema.Schema{Type: schema.TypeString},
+				ForceNew: true,
+			},
+			"output_file": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+
+			// Computed values
+			"bandwidth_limits": {
```
Comment (xiaozhu36) [inline alicloud/data_source_alicloud_cen_bandwidth_limits.go]: please use `limits` to instead `bandwidth_limits `

File: alicloud/data_source_alicloud_cen_bandwidth_limits_test.go [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+    instance_id = "${alicloud_cen_instance.cen.id}"
+    child_instance_id = "${alicloud_vpc.vpc2.id}"
+    child_instance_region_id = "cn-shanghai"
+}
+
+resource "alicloud_cen_bandwidth_limit" "foo" {
+     instance_id = "${alicloud_cen_instance.cen.id}"
+     region_ids = [
+					"cn-beijing",
+					"cn-shanghai"]
```
Comment (xiaozhu36) [inline alicloud/data_source_alicloud_cen_bandwidth_limits_test.go]: please format it.

File: website/docs/d/cen_bandwidth_limits.html.markdown [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+layout: "alicloud"
+page_title: "Alicloud: alicloud_cen_bandwidth_limits"
+sidebar_current: "docs-alicloud-datasource-cen-bandwidth-limits"
+description: |-
+    Provides a list of CEN Inter Region Bandwidth Limits owned by an Alibaba Cloud account.
+---
+
+# alicloud\_cen\_bandwidth\_limits
+
+This data source provides CEN Inter Region Bandwidth Limits available to the user.
```
Comment (xiaozhu36) [inline website/docs/d/cen_bandwidth_limits.html.markdown]: What is `Inter Region`?

Comment (GiantRice) [reply website/docs/d/cen_bandwidth_limits.html.markdown]: It means cross-region.  It seems redundant here. I'll remove it .

**Pull request merged**